bubblesub/model.py

```diff
@@ -20,7 +20,7 @@
 
 from PyQt5 import QtCore
 
-TItem = T.TypeVar("TItem")  # pylint: disable=invalid-name
```

Comment:
What happened to this?

Reply:
Don't know, but it seems to work without that disabler